dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,487 @@
+
+### Configuration
+
+Use `setConfig` to instruct Prebid.js to initilize the exadsBidAdapter, as specified below. 
+* Set "debug" as true if you need to read logs;
+* Set "gdprApplies" as true if you need to pass gdpr consent string;
+* The tcString is the iabtcf consent string for gdpr;
+* Uncomment the cache instruction if you need to configure a cache server (e.g. for instream video)
+
+```
+pbjs.setConfig({
+    debug: false,
+    //cache: { url: "https://prebid.adnxs.com/pbc/v1/cache" },
+    consentManagement: {
+        gdpr: {
+            cmpApi: 'static',
+            timeout: 1000000,
+            defaultGdprScope: true,
+            consentData: {
+                getTCData: {
+                    tcString: consentString,
+                    gdprApplies: false // set to true to pass the gdpr consent string
+                }
+            }
+        }
+    }
+});
+```
+Add the `video` config if you need to render videos using the video module.
+For more info navigate to https://docs.prebid.org/prebid-video/video-module.html.
+```
+pbjs.setConfig({
+    video: {
+        providers: [{
+            divId: 'player', // the id related to the videojs tag in your body
+            vendorCode: 2, // videojs, 
+            playerConfig: {
+                params: {
+                    adPluginConfig: {
+                        numRedirects: 10
+                    },
+                    vendorConfig: {
+                        controls: true,
+                        autoplay: true,
+                        preload: "auto",
+                    }
+                }
+            }
+        },]
+    },
+});
+```
+
+#### RTB Banner 2.4
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description           | Example   | Type      |
+|---------------|----------|-----------------------|-----------|-----------|
+| `zoneID`      | required | You can get it from the endpoint created after configuring the zones          | `12345`    | `'integer'` |
+| `fid`      | required | You can get it from the endpoint created after configuring the zones          | `'829a896f011475d50da0d82cfdd1af8d9cdb07ff'`    | `'string'` |
+| `partner`      | required | Currently we support rtb 2.4 (“ortb_2_4”) only           | `'ortb_2_4'`    | `'string'` |
+| `siteId`      | recommended |  Unique Site ID           | `'123'`    | `'string'` |
+| `siteName`      |  |  Site name            | `'test.com'`    | `'string'` |
+| `banner.sizes`      | required |  [width, height]          | `[145,256]`    | `'integer array'` |
+| `userIp`      | required |  IP address of the user, ipv4 or ipv6          | `'0.0.0.0'`    | `'string'` |
+| `userId`      | *required |  Unique user ID (string). *If you cannot generate a user ID, you can leave it empty (""). The request will get a response as long as “user” object is included in the request          | `''`    | `'string'` |
+| `country`      |  |  country ISO3          | `'IRL'`    | `'string'` |
```

Comment:
@giuseppe-exads Please specify then it's optional by writing it in the scope section.

Reply:
done